tests/backends/models.py

```diff
@@ -113,3 +113,18 @@ def __str__(self):
 
 class RawData(models.Model):
     raw_data = models.BinaryField()
+
+# We need these models only for Oracle DB tests
+if connection.vendor == 'oracle':
+    class Student(models.Model):
+        name = models.CharField(max_length=30)
+
+        class Meta:
+            db_table = '"test_%s"."DJANGO_STUDENT_TABLE"' % (connection.settings_dict['USER'])
```

Comment:
You can override test username in settings. Please use `connection.settings_dict['TEST']['USER']`:

```python
db_table = '"%s"."DJANGO_STUDENT_TABLE"' % (connection.settings_dict['TEST']['USER'])
```